test_azfilesfuse.py

```diff
@@ -55,7 +54,7 @@ def cdll_mock(*args, **kwargs):
 class Test_azfilesfuse(unittest.TestCase):
     STORAGE_ACCOUNT_NAME='crwilcoxmsftplayground'
     STORAGE_ACCOUNT_SHARE='fusetests'
-    STORAGE_ACCOUNT_SAS_TOKEN=None
+    STORAGE_ACCOUNT_SAS_TOKEN='None'
```

Comment:
Did you mean for this change?

Reply:
It was accidental at first, but I ended up being pretty OK with the pattern of stubbing all the fields the same way (as strings).

LMK if you feel strongly about this, I can change it back.

Reply:
That is fine. I don't really mind but didn't see the reasoning. I think the logic is to prefer the env vars over this anyway. Though I think this may prevent throwing an error saying it wasn't set?